spec/formatting.md

```diff
@@ -230,65 +231,54 @@ the following steps are taken:
 
 ### Fallback Resolution
 
-A **_fallback value_** is the resolved value emitted when an _expression_ cannot be resolved.
+A **_fallback value_** is the resolved value for an _expression_ that fails to resolve.
 
 An _expression_ fails to resolve when:
 
-- A _variable_ _operand_ fails to resolve.
+- A _variable_ used as complete _expression_ contents or as an _operand_ fails to resolve.
+  * Note that this does not include a _variable_ used as an _option_ value.
 - A _function_ _annotation_ fails to resolve.
 - A _private-use_ _annotation_ is unsupported by the implementation or if
   a _private-use_ _annotation_ fails to resolve.
 - The _expression_ has a _reserved_ _annotation_.
 
 The _fallback value_ depends on the contents of the _expression_:
 
-- _expression_ with _literal_ _operand_: U+007C VERTICAL LINE `|`
-  followed by the value of the Literal,
-  and then by U+007C VERTICAL LINE `|`.
-  The same representation is used for both _quoted_ and _unquoted_ values.
-
-  > Examples: `|your horse|`, `|42|`
+- _expression_ with _annotation_ and no _operand_:
+  the _annotation_ starting sigil followed by its _identifier_
 
-- _expression_ with _variable_ _operand_: U+0024 DOLLAR SIGN `$`
-  followed by the _variable_ _name_ of the _operand_
+  > Examples:
+  > In a context where `:func` fails to resolve, `{:func}` resolves to the _fallback value_ `:func`.
+  > `{@reserved |...|}` always resolves to the _fallback value_ `@reserved`.
 
-  > Example: `$user`
+- _expression_ with _literal_ _operand_:
+  U+007C VERTICAL LINE `|`
+  followed by the value of the _literal_
+  with escaping applied to U+005C REVERSE SOLIDUS `\` and U+007C VERTICAL LINE `|`,
+  and then by U+007C VERTICAL LINE `|`.
+  The same representation is used for both _quoted_ and _unquoted_ values.
 
-- _expression_ with no _operand_:
-  the _function_ starting sigil followed by its _name_
+  > Examples:
+  > In a context where `:func` fails to resolve,
+  > `{42 :func}` resolves to the _fallback value_ `|42|` and
+  > `{|C:\\| :func}` resolves to the _fallback value_ `|C:\\|`.
+  > `{|| @reserved}` always resolves to the _fallback value_ `||`.
```

Comment:
The fallback loses the `{`/`}`

Reply:
The braces are added outside of these steps, cf. [Formatting Fallback Values](https://github.com/unicode-org/message-format-wg/blob/main/spec/formatting.md#formatting-fallback-values). I've updated the relevant examples to clarify.